src/worker/keyman-api-service.ts

```diff
@@ -0,0 +1,36 @@
+import { keyboardDataObj, KeymanObj } from "./models";
+
+export class KeymanApi {
+  /**
+   * API configs
+   */
+  baseUrl = "https://api.keyman.com/search";
+  /**
+   * Query data to searche only for languages that match
+   */
+  params = { q: "l" };
```

Comment:
Could you explain how this query will return the results of, say the "Halq'eméylem" keyboard layout for mobile?

Reply:
this query will return all existing keyboard

Reply:
After reading the documentation, I feel like `?q=l` returns all keyboards whose languages have an "l" in the name. I think `?q=` will return ALL keyboards.